maven-core/src/main/java/org/apache/maven/project/ProjectSorter.java

```diff
@@ -105,7 +105,7 @@ public ProjectSorter( Collection<MavenProject> projects )
             vertices.put( project.getVersion(), dag.addVertex( projectId ) );
         }
 
-        for ( Vertex projectVertex : dag.getVerticies() )
+        for ( Vertex projectVertex : dag.getVertices() )
```

Comment:
this change stands alone, I can't find anything else that uses this spelling, but lots of things use the corrected spelling....